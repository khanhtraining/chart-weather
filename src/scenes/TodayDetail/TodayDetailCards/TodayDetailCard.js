import React from 'react'
import './todaydetailcards.scss'

const TodayDetailCard = (props) => {
    const { title, content, className, number, classTitle, classContent } = props

    return (
        <React.Fragment>
            <div className='todayDetailCards-wrapper-item'>
                <div className={classTitle}>{title}</div>
                <div className={`my class ${className}`}>{number}</div>
                <div className={classContent}>{content}</div>
            </div>
        </React.Fragment>
    )
}

export default TodayDetailCard
