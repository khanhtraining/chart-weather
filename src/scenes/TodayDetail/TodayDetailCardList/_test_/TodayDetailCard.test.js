import React from 'react'
import renderer from 'react-test-renderer'
import TodayDetailCard from '../TodayDetailCard'

describe('snapshot testing', () => {
    test('snapshot for today detail card component', () => {
        const renderedComponent = renderer.create(<TodayDetailCard/>).toJSON()
        expect(renderedComponent).toMatchSnapshot()
    })
})
