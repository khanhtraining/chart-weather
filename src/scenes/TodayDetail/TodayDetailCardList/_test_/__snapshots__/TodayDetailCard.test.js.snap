// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing snapshot for Today Detail Card component 1`] = `
<div
  className="todayDetailCards-wrapper-item"
>
  <div
    className="title"
  />
  <div
    className="content undefined"
  />
  <div
    className="extra"
  />
</div>
`;

exports[`snapshot testing snapshot for Today Detail component 1`] = `
<div
  className="todayDetailCards-wrapper-item"
>
  <div
    className="title"
  />
  <div
    className="content undefined"
  />
  <div
    className="extra"
  />
</div>
`;

exports[`snapshot testing snapshot for today detail card component 1`] = `
<div
  className="todayDetailCards-wrapper-item"
>
  <div
    className="title"
  />
  <div
    className="content undefined"
  />
  <div
    className="extra"
  />
</div>
`;
